src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -602,11 +601,12 @@ The [Cognito Hosted UI]( https://docs.aws.amazon.com/cognito/latest/developergui
 After you test the sign-up, sign-in, and sign-out functionality, you can take a look at the created user in the Amazon Cognito console. To do so, you can use `amplify console auth` to open the console directly from the CLI, or take a look at the existing User Pools in the console. Note the users status should be "CONFIRMED" if you successfully verified the user; however, it can be in "unconfirmed" if the user is not yet verified. You can also check to make sure any other required attributes are updating correctly.
 
 **Recap:** You now have enabled sign-up, sign-in, and sign-out for your authentication and verified its functionality with a test user.
-
+ 
 ## Conclusion
 
 Congratulations! You finished the **Set up Amplify Auth** guide. In this guide, you set up and connected to backend resources, compared and determined your Auth integration path, and enabled sign-up, sign-in, and sign-out while verifying they function as expected.
 
 ## Next steps
 
 TBD - Potentially a link to Social Sign In and MFA setup.
+ 
```

Comment:
Reminder for us to fill in this TBD with clear call to action links and why they matter - original comment - Provide 2–3 relevant next steps. These can be links related to what was covered in the guide and/or links to related Amplify jobs-to-be-done guides.

Reply:
This may change again with the revisions planned as we may point to removed items here but filling in this as a placeholder based on the previous recommendation.